### PR TITLE
Add DDF for Aqara H1 switches w/o neutral WS-EUK01/WS-EUK02

### PR DESCRIPTION
**Pull request by SwoopX**
- Adds the respective DDFs and removes the associated legacy code
- Prevent ghost button events for Aqara H1 switch series

### CODE REVIEW

File: de_web_plugin.cpp [hunk 1]
```diff
@@ -5105,13 +5102,13 @@ void DeRestPluginPrivate::checkSensorButtonEvent(Sensor *sensor, const deCONZ::A
                     quint8 dataType;
```
[the remark this answers is not in the document]
Comment (manup) [reply de_web_plugin.cpp:5102]: Little test with the PR and Xiaomi WXKG01LM (the round button), seems to break only 1000 is detected and something weird with 1001 but the multi-press events aren't triggered anymore.

**Comment by manup** [comment]
I think this needs some work to not break other switches, see comments in file section.

File: de_web_plugin.cpp (hunk 1 above)
[the remark this answers is not in the document]
Comment (SwoopX) [reply de_web_plugin.cpp]: Pandorra's box... Solved it differently now.

**Comment by manup** [comment]
Thanks, that should do the trick. I'll test later today with my older Xiaomi switches.

**Comment by manup** [comment]
Tested a bunch of switches, looks fine :+1:

**Pull request merged**